Use the new octokit

We ran into an error on `github@14.0.0`

```
$ npm run collect

> electron-i18n@0.0.0-development collect /Users/z/git/electron/i18n
> node script/collect.js

/Users/z/git/electron/i18n/node_modules/github/index.js:1
(function (exports, require, module, __filename, __dirname) { throw new Error("'github' has been renamed to '@octokit/rest' (https://git.io/vNB11)")
                                                              ^

Error: 'github' has been renamed to '@octokit/rest' (https://git.io/vNB11)
    at Object.<anonymous> (/Users/z/git/electron/i18n/node_modules/github/index.js:1:69)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/z/git/electron/i18n/script/collect.js:14:16)
    at Module._compile (module.js:652:30)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! electron-i18n@0.0.0-development collect: `node script/collect.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the electron-i18n@0.0.0-development collect script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/z/.npm/_logs/2018-05-02T16_46_21_502Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -10,6 +10,7 @@
   "author": "zeke",
   "license": "MIT",
   "devDependencies": {
+    "@octokit/rest": "^15.2.7",
     "chai": "^4.1.2",
     "chai-date-string": "^0.1.0",
     "check-for-leaks": "^1.2.0",
@@ -20,7 +21,6 @@
     "electron-docs": "^3.0.2",
     "form-data": "^2.3.2",
     "get-crowdin-file-ids": "^1.1.1",
-    "github": "13.0.0",
     "github-slugger": "^1.2.0",
     "globals": "^11.3.0",
     "got": "^8.3.0",

--- a/script/collect.js
+++ b/script/collect.js
@@ -9,14 +9,15 @@ const got = require('got')
 const mkdir = require('make-dir').sync
 const path = require('path')
 const {execSync} = require('child_process')
-
+const github = require('@octokit/rest')()
 const englishBasepath = path.join(__dirname, '..', 'content', 'en-US')
-const GitHub = require('github')
-const github = new GitHub({
-  debug: false,
-  Promise: Promise,
-  token: process.env.GH_TOKEN
-})
+
+if (process.env.GH_TOKEN) {
+  github.authenticate({
+    type: 'token',
+    token: process.env.GH_TOKEN
+  })
+}
 
 let release
 

